src/main/java/app/entities/Account.java

```diff
@@ -0,0 +1,62 @@
+package app.entities;
+
+public class Account {
+    private int accountId;
+    private String name;
+    private String address;
+    private String city;
```

Comment:
Jeg kan godt lide at ting kommer i en logisk rækkefølge (så vidt det er muligt).
Så zip først, og dernæst city, ligesom at man jo skriver "2100 København Ø"
(Måske er det bare mig der er for detaljeorienteret)